unified-codes/libs/data/src/lib/types/EntitySearch.ts

```diff
@@ -0,0 +1,12 @@
+import { EEntityType } from './Entity';
+
+import IEntitySort from './EntitySort';
+
+export interface IEntitySearch {
```

Comment:
Getting better at less off-roading! Minor name change here, just to make it a bit more clear what this type is for!